service/src/test/java/com/sedliarov/learningtable/integration/StudentControllerIntegrationTests.java

```diff
@@ -22,7 +22,6 @@
  *
  * @author Kirill Sedliarov
  */
-
 public class StudentControllerIntegrationTests extends RestIntegrationTestBase {
```

Comment:
Maybe add some changes for tests because for now we can expect Error with defined message)